### PR TITLE
libgda 5.2.4

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/libgda.rb [hunk 1]
```diff
@@ -3,8 +3,9 @@
 class Libgda < Formula
   desc "Provides unified data access to the GNOME project"
   homepage 'http://www.gnome-db.org/'
-  url 'http://ftp.gnome.org/pub/GNOME/sources/libgda/5.2/libgda-5.2.0.tar.xz'
-  sha256 '41bd14aaaf50efc7b80d7279c69ed9c90d3a1894cb5123385d86883a1d7d5f30'
+  url 'http://ftp.gnome.org/pub/GNOME/sources/libgda/5.2/libgda-5.2.4.tar.xz'
```
Comment (bfontaine) [inline Library/Formula/libgda.rb]: Please use `https://download.gnome.org/sources/libgda/5.2/libgda-5.2.4.tar.xz` instead

Comment (nijikon) [inline Library/Formula/libgda.rb]: @bfontaine ok

**Comment by nijikon** [comment]
@bfontaine done

**Comment by mikemcquaid** [comment]
Thanks @nijikon!

**Pull request closed without merging**